Real Phonon headers are installed to the parent directory rather
than current directory of these class headers.
--- a/include/Phonon/AbstractAudioOutput
+++ b/include/Phonon/AbstractAudioOutput
@@ -1 +1 @@
-#include "abstractaudiooutput.h"
+#include "../abstractaudiooutput.h"
--- a/include/Phonon/AbstractMediaStream
+++ b/include/Phonon/AbstractMediaStream
@@ -1 +1 @@
-#include "abstractmediastream.h"
+#include "../abstractmediastream.h"
--- a/include/Phonon/AbstractVideoOutput
+++ b/include/Phonon/AbstractVideoOutput
@@ -1 +1 @@
-#include "abstractvideooutput.h"
+#include "../abstractvideooutput.h"
--- a/include/Phonon/AddonInterface
+++ b/include/Phonon/AddonInterface
@@ -1 +1 @@
-#include "addoninterface.h"
+#include "../addoninterface.h"
--- a/include/Phonon/AudioCaptureDevice
+++ b/include/Phonon/AudioCaptureDevice
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/AudioCaptureDeviceModel
+++ b/include/Phonon/AudioCaptureDeviceModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/AudioChannelDescription
+++ b/include/Phonon/AudioChannelDescription
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/AudioChannelDescriptionModel
+++ b/include/Phonon/AudioChannelDescriptionModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/AudioOutput
+++ b/include/Phonon/AudioOutput
@@ -1 +1 @@
-#include "audiooutput.h"
+#include "../audiooutput.h"
--- a/include/Phonon/AudioOutputDevice
+++ b/include/Phonon/AudioOutputDevice
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/AudioOutputDeviceModel
+++ b/include/Phonon/AudioOutputDeviceModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/AudioOutputInterface
+++ b/include/Phonon/AudioOutputInterface
@@ -1 +1 @@
-#include "audiooutputinterface.h"
+#include "../audiooutputinterface.h"
--- a/include/Phonon/AudioOutputInterface40
+++ b/include/Phonon/AudioOutputInterface40
@@ -1 +1 @@
-#include "audiooutputinterface.h"
+#include "../audiooutputinterface.h"
--- a/include/Phonon/AudioOutputInterface42
+++ b/include/Phonon/AudioOutputInterface42
@@ -1 +1 @@
-#include "audiooutputinterface.h"
+#include "../audiooutputinterface.h"
--- a/include/Phonon/BackendCapabilities
+++ b/include/Phonon/BackendCapabilities
@@ -1 +1 @@
-#include "backendcapabilities.h"
+#include "../backendcapabilities.h"
--- a/include/Phonon/BackendInterface
+++ b/include/Phonon/BackendInterface
@@ -1 +1 @@
-#include "backendinterface.h"
+#include "../backendinterface.h"
--- a/include/Phonon/Effect
+++ b/include/Phonon/Effect
@@ -1 +1 @@
-#include "effect.h"
+#include "../effect.h"
--- a/include/Phonon/EffectDescription
+++ b/include/Phonon/EffectDescription
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/EffectDescriptionModel
+++ b/include/Phonon/EffectDescriptionModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/EffectInterface
+++ b/include/Phonon/EffectInterface
@@ -1 +1 @@
-#include "effectinterface.h"
+#include "../effectinterface.h"
--- a/include/Phonon/EffectParameter
+++ b/include/Phonon/EffectParameter
@@ -1 +1 @@
-#include "effectparameter.h"
+#include "../effectparameter.h"
--- a/include/Phonon/EffectWidget
+++ b/include/Phonon/EffectWidget
@@ -1 +1 @@
-#include "effectwidget.h"
+#include "../effectwidget.h"
--- a/include/Phonon/Global
+++ b/include/Phonon/Global
@@ -1 +1 @@
-#include "phononnamespace.h"
+#include "../phononnamespace.h"
--- a/include/Phonon/MediaController
+++ b/include/Phonon/MediaController
@@ -1 +1 @@
-#include "mediacontroller.h"
+#include "../mediacontroller.h"
--- a/include/Phonon/MediaNode
+++ b/include/Phonon/MediaNode
@@ -1 +1 @@
-#include "medianode.h"
+#include "../medianode.h"
--- a/include/Phonon/MediaObject
+++ b/include/Phonon/MediaObject
@@ -1 +1 @@
-#include "mediaobject.h"
+#include "../mediaobject.h"
--- a/include/Phonon/MediaObjectInterface
+++ b/include/Phonon/MediaObjectInterface
@@ -1 +1 @@
-#include "mediaobjectinterface.h"
+#include "../mediaobjectinterface.h"
--- a/include/Phonon/MediaSource
+++ b/include/Phonon/MediaSource
@@ -1 +1 @@
-#include "mediasource.h"
+#include "../mediasource.h"
--- a/include/Phonon/ObjectDescription
+++ b/include/Phonon/ObjectDescription
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/ObjectDescriptionData
+++ b/include/Phonon/ObjectDescriptionData
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/ObjectDescriptionModel
+++ b/include/Phonon/ObjectDescriptionModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/ObjectDescriptionModelData
+++ b/include/Phonon/ObjectDescriptionModelData
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/Path
+++ b/include/Phonon/Path
@@ -1 +1 @@
-#include "path.h"
+#include "../path.h"
--- a/include/Phonon/PlatformPlugin
+++ b/include/Phonon/PlatformPlugin
@@ -1 +1 @@
-#include "platformplugin.h"
+#include "../platformplugin.h"
--- a/include/Phonon/SeekSlider
+++ b/include/Phonon/SeekSlider
@@ -1 +1 @@
-#include "seekslider.h"
+#include "../seekslider.h"
--- a/include/Phonon/StreamInterface
+++ b/include/Phonon/StreamInterface
@@ -1 +1 @@
-#include "streaminterface.h"
+#include "../streaminterface.h"
--- a/include/Phonon/SubtitleDescription
+++ b/include/Phonon/SubtitleDescription
@@ -1 +1 @@
-#include "objectdescription.h"
+#include "../objectdescription.h"
--- a/include/Phonon/SubtitleDescriptionModel
+++ b/include/Phonon/SubtitleDescriptionModel
@@ -1 +1 @@
-#include "objectdescriptionmodel.h"
+#include "../objectdescriptionmodel.h"
--- a/include/Phonon/VideoPlayer
+++ b/include/Phonon/VideoPlayer
@@ -1 +1 @@
-#include "videoplayer.h"
+#include "../videoplayer.h"
--- a/include/Phonon/VideoWidget
+++ b/include/Phonon/VideoWidget
@@ -1 +1 @@
-#include "videowidget.h"
+#include "../videowidget.h"
--- a/include/Phonon/VideoWidgetInterface
+++ b/include/Phonon/VideoWidgetInterface
@@ -1 +1 @@
-#include "videowidgetinterface.h"
+#include "../videowidgetinterface.h"
--- a/include/Phonon/VolumeFaderEffect
+++ b/include/Phonon/VolumeFaderEffect
@@ -1 +1 @@
-#include "volumefadereffect.h"
+#include "../volumefadereffect.h"
--- a/include/Phonon/VolumeFaderInterface
+++ b/include/Phonon/VolumeFaderInterface
@@ -1 +1 @@
-#include "volumefaderinterface.h"
+#include "../volumefaderinterface.h"
--- a/include/Phonon/VolumeSlider
+++ b/include/Phonon/VolumeSlider
@@ -1 +1 @@
-#include "volumeslider.h"
+#include "../volumeslider.h"
